pkg/core/transaction/type.go

```diff
@@ -51,9 +51,5 @@ func (t TXType) String() string {
 
 // MarshalJSON implements the json marshaller interface.
 func (t TXType) MarshalJSON() ([]byte, error) {
-	j, err := json.Marshal(t.String())
-	if err != nil {
-		return nil, err
-	}
-	return j, nil
+	return json.Marshal(t.String())
```

Comment:
more fast and simple way
```go
func (t TXType) MarshalJSON() ([]byte, error) {
	return []byte(`"` + t.String() + `"`), nil
}
```

Reply:
as I proposed [here](https://github.com/CityOfZion/neo-go/pull/135#discussion_r257503551)

Reply:
@im-kulikov , I believe you are not looking at the latest commit. Your suggestion is already implemented there.

Reply:
nope.. that was first solution.. more fast and simple is above

Reply:
@dauTT https://github.com/CityOfZion/neo-go/pull/135/files#diff-7ece7ccdd490e483b6bce3566e70297fR54

and:
```go
func (t TXType) MarshalJSON() ([]byte, error) {
	return []byte(`"` + t.String() + `"`), nil
}
```

Reply:
there is no sense to call `json.Marshal` for string

Reply:
:+1: